### ISSUE TITLE
endianness for sequential/rijndael_xxx

### ISSUE THREAD

**Issue by flopsi** [tacle/tacle-bench#8]
The code for sequential/rijndael_enc and sequential/rijndael_dec appears to be sensitive to the platform's endianness. The AES source code from MiBench, for instance, contains explicit conversions to handle this.

Simple Solution:
A simple solution would be to introduce defines that control endianess (which would be needed for other benchmarks anyways -- more issue requests are on the way). This would allow at least to compare against checksums depending on the platform's endianness.

**Comment by flopsi** [comment]
Btw., the same applies also to sequential/adpcm_enc.

**Comment by schoeberl** [comment]
Yes, this is known that there are some endianess dependencies in the benchmarks. I consider this bad C code and argued for removing those benchmarks. And I still am ;-)
